.github/workflows/job-precommit-python-avro-io.yml

```diff
@@ -0,0 +1,73 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This workflow will run the test of python Avro IO class.
+
+name: Python Avro IO
```

Comment:
Do we consider AvroIO as part of "core IO" like in Java SDK so this test may be exercised more often (as textio)? CC: @chamikaramj may have more context.

Also, there is not much precommit test for hadoop, we may not need to split it at this point.

Reply:
I removed the Hadoop precommit suite, those tests will be running in the remaining python precommit suite in https://github.com/apache/beam/pull/23451